specs/template.md

```diff
@@ -1,33 +1,63 @@
-<!-- USAGE
-  * Fill in each of the sections (like Background) below
-  * Wrap code with `single line of code` or ```code block```
-  * Before submitting, delete all <!-- TEMPLATE marked comments in this file,
+<!-- 
+    Before submitting, delete all "<!-- TEMPLATE marked" comments in this file,
```

Comment:
nit: the quotation mark shouldn't include the work "marked"

Reply:
Fixed thx